src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -31,14 +31,14 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.awaitility.Awaitility.await;
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.BULK_SCANNED_CASE_TYPE;
-import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdNewCaseCreator.EXCEPTION_RECORD_REFERENCE;
```

Comment:
This constant doesn't exist anymore. Replaced by a constant from `ServiceCaseFields` class (dedicated to constants).